Bmx.CommandLine/CommandLine.cs

```diff
@@ -14,7 +15,7 @@ public class CommandLine {
 
 		public CommandLine() {
 			// TODO: Fix this :| (ref: Program.cs)
-			_bmx = new BmxCore( new OktaClient( "d2l" ) );
+			_bmx = new BmxCore( new OktaClient( "d2l" ), new AwsClient() );
```

Comment:
I assume the `TODO` here is to inject these instead of using `new` and maybe move `"d2l"` to some sort of config?

Reply:
It is a required parameter of bmx (from 2.0) so from launch args rather than config.

Constructor makes it mandatory to be set, my other alternative is to move setting Organization to some where after the OktaClient is initialized, it seems like a bit of a code smell since there is no way to enforce that organization is set prior to use of this class.

Reply:
> It is a required parameter of bmx (from 2.0) so from launch args rather than config.

Not sure I understand you correctly, but I'm pretty sure in bmx 2+ you can still set org in config.

Reply:
ATM this rewrite doesn't have read from config yet.
When there isn't a config present its a required parameter with BMX 2.0. 

Regardless of how its set and read in it won't be possible to access organization to pass onto the constructor here so I would need to move the logic to set organization elsewhere.

Reply:
Also recall that this is an open source project - there shouldn't be any reference to `d2l` in the code.

Reply:
> Regardless of how its set and read in it won't be possible to access organization to pass onto the constructor here so I would need to move the logic to set organization elsewhere.

What I normally do is to read the config in the constructor. The config can be composed of different sources including config files and command line args.
More here: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1

Reply:
Would like to handle proper loading of configs with #191 so different PR.